go/vt/wrangler/schema.go

```diff
@@ -56,31 +57,35 @@ func (wr *Wrangler) ReloadSchema(ctx context.Context, tabletAlias *topodatapb.Ta
 // and the periodic schema reload makes them self-healing anyway.
 // So we do this on a best-effort basis, and log warnings for any tablets
 // that fail to reload within the context deadline.
-// Note that this skips the master because it's assumed that the schema
-// was reloaded there right after applying it.
-func (wr *Wrangler) ReloadSchemaShard(ctx context.Context, keyspace, shard, replicationPos string) {
+func (wr *Wrangler) ReloadSchemaShard(ctx context.Context, keyspace, shard, replicationPos string, concurrency *sync2.Semaphore, includeMaster bool) {
 	tablets, err := wr.ts.GetTabletMapForShard(ctx, keyspace, shard)
-	if err != nil {
-		if err != topo.ErrPartialResult {
-			// This is best-effort, so just log it and move on.
-			wr.logger.Warningf("Failed to reload schema on slave tablets in %v/%v (use vtctl ReloadSchema to fix individual tablets): %v", keyspace, shard, err)
-			return
-		}
-		// We got a partial result. Do what we can, but warn that some may be missed.
+	switch err {
+	case topo.ErrPartialResult:
+		// We got a partial result. Do what we can, but warn
+		// that some may be missed.
 		wr.logger.Warningf("Failed to fetch all tablets for %v/%v. Some slave tablets may not have schema reloaded (use vtctl ReloadSchema to fix individual tablets)", keyspace, shard)
+	case nil:
+		// Good case, keep going too.
+	default:
+		// This is best-effort, so just log it and move on.
+		wr.logger.Warningf("Failed to reload schema on slave tablets in %v/%v (use vtctl ReloadSchema to fix individual tablets): %v", keyspace, shard, err)
```

Comment:
a) I think the erorr message is misleading. It's not that reload schema failed. Instead, loading all tablets from topology failed and you're operating on a subset now.
b) You should mention ReloadSchemaShard as well because it will be easier than singling out the failed tablets.